### PR TITLE
Improve icons

### PR DESCRIPTION
**Pull request by ghost**
* Add .git folder icon. Fix https://github.com/manuelbieh/logo-file-icons/issues/8
* Fix tab's icon
* Remove files and notes
* New eslint icon

### CODE REVIEW

File: styles/logo-file-icons.less [hunk 1]
```diff
@@ -184,77 +174,26 @@
...
 
-  span[data-name=".npmrc"]:before,
-  span[data-name="npm-debug.log"]:before,
-  span[data-name=".npmignore"]:before,
-  div[data-name$="package.json"]:before,
-  span[data-name$="package.json"]:before { .file-icon(npm); }
+  [data-name=".npmrc"]:before,
+  [data-name="npm-debug.log"]:before,
+  [data-name=".npmignore"]:before,
+  [data-name="package.json"]:before { .file-icon(npm); }
```
Comment (manuelbieh) [inline styles/logo-file-icons.less:179]: hmm you're sure this is working? I have added the `span` to the selector to give it a higher specificity, since `[data-name="package.json"]` was "overwritten" by `[data-name$=".json"]`.

File: styles/logo-file-icons.less [hunk 2]
```diff
@@ -184,77 +174,26 @@
...
 
-  span[data-name$="bower.json"]:before { .file-icon(bower); }
-  span[data-name^="composer."]:before { .file-icon(composer); }
+  [data-name="bower.json"]:before { .file-icon(bower); }
+  [data-name="composer.json"]:before { .file-icon(composer); }
```
Comment (manuelbieh) [inline styles/logo-file-icons.less:185]: I left out the `json` on intention to also cover `composer.lock`. Can you revert this change or at least add `composer.lock` as separate rule please?

File: styles/logo-file-icons.less [hunk 3]
```diff
@@ -184,77 +174,26 @@
...
 
-  span[data-name$="Gruntfile.js"]:before { .file-icon(grunt); }
-  span[data-name$="gruntfile.js"]:before { .file-icon(grunt); }
+  [data-name^="Gruntfile"]:before { .file-icon(grunt); }
```
Comment (manuelbieh) [inline styles/logo-file-icons.less]: Change it to 
```
[data-name^="Gruntfile"]:before,
[data-name^="gruntfile"]:before { .file-icon(grunt); }
```
... just like you did with Gulpfile below to keep it consistent 😃

File: styles/logo-file-icons.less [hunk 4]
```diff
@@ -184,77 +174,26 @@
...
+  [data-name^="gruntfile"]:before { .file-icon(grunt); }
 
-  span[data-name^="Gulpfile"]:before { .file-icon(gulp); }
-  span[data-name^="gulpfile"]:before { .file-icon(gulp); }
-
-  span[data-name$=".pdf"]:before { .file-icon(pdf); }
+  [data-name^="Gulpfile"],
```
Comment (manuelbieh) [inline styles/logo-file-icons.less]: `[data-name^="Gulpfile"]:before` – the `:before` is missing here :wink:

File: README.md [hunk 5]
```diff
@@ -1,17 +1,15 @@
...
-Add some nice logo icons to your files in Atom. Logos were taken from http://svgporn.com/
+Add some nice logo icons to your files in Atom. Logos were taken from [SVGPORN](http://svgporn.com)
 
 All logos are the property of their respective owners.
 
 ## Preview
 
 ### Light theme
-![Preview light](https://raw.githubusercontent.com/manuelbieh/logo-file-icons/master/logo-file-icons.png)
+![Preview-light](https://cloud.githubusercontent.com/assets/16760489/21432324/931e4c9a-c89d-11e6-9309-6707ecdaddb9.png)
```
Comment (manuelbieh) [inline README.md]: Since you are adding the new images here you can leave the url as it is 😉

File: README.md [hunk 6]
```diff
@@ -1,17 +1,15 @@
...
 
 ### Dark theme
-![Preview dark](https://raw.githubusercontent.com/manuelbieh/logo-file-icons/master/logo-file-icons-dark.png)
-
-
+![Preview dark](https://cloud.githubusercontent.com/assets/16760489/21432323/931b6d0e-c89d-11e6-99cd-a14d6640696f.png)
```
Comment (manuelbieh) [inline README.md:13]: Here as well ...

**Review by manuelbieh — CHANGES_REQUESTED** [review]

**Comment by manuelbieh** [comment]
Thanks a lot for your great PR. I've done some reviewing and found a few "bugs" which need to be fixed before I can merge it.

Don't forget to add yourself to the list of contributors in the package.json if you like :wink:

File: styles/logo-file-icons.less (hunk 1 above)
Comment (ghost) [reply styles/logo-file-icons.less:179]: I can sure about this. You can see new images.

File: styles/logo-file-icons.less (hunk 2 above)
Comment (torn4dom4n) [inline styles/logo-file-icons.less]: Yeah!

File: README.md (hunk 5 above)
Comment (ghost) [reply README.md]: I don't understand this. Do it mean I should upload an images to logo-files-icons?

Comment (ghost) [reply README.md]: I read some README file like [setings-view](https://github.com/atom/settings-view), [markdown-preview](https://github.com/atom/markdown-preview) so I think it shoulds change.

**Review by manuelbieh — APPROVED** [review]

**Comment by ghost** [comment]
🎉
